### PR TITLE
Welcome to Portland post

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

**Comment by plaindocs** [comment]
FIXED  ~The heading looks broke on all the news posts. I guess it is a path issue?~

File: docs/conf/portland/2018/news/welcome-to-portland.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+* **Job Fair**. New this year, on Tuesday morning we'll have a `job fair <http://www.writethedocs.org/conf/portland/2018/job-fair>`_ where you can find your next documentarian friendly job.
+* **Plan a discussion.** We'll be running the `Unconference <http://www.writethedocs.org/conf/portland/2018/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a very specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
+* **Coffee, breakfast, and lunch are provided on all three days!** Don't worry about coffee before you come to the conference on Sunday, Monday, or Tuesday. Coffee and breakfast are served at 8:15, and the conference starts at 9. As a part of our ongoing effort to make every aspect of the event friendly and inclusive (that includes the food too!), the full catering menu with allergen indications will be available on our `Venue page <http://www.writethedocs.org/conf/portland/2018/venue/#dietary-requirements>`_.
```
Comment (ericholscher) [inline docs/conf/portland/2018/news/welcome-to-portland.rst]: I don't think we actually put the menu up this year. It's basically the same as last year though, so maybe we can just repurpose that.

File: docs/conf/portland/2018/news/welcome-to-portland.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Find Your Way Around
+--------------------
+
+* **Riding the trains.** Be aware that TriMet, Portland's public transit system, will be working on the light rail lines. This means that you can't easily get from the Airport to downtown without a transfer. Public transit is generally excellent in Portland, but trains might be crowded, and schedules will be limited. For more information, see our `visiting page <http://www.writethedocs.org/conf/portland/2018/visiting/#the-max>`_ and the `TriMet website <https://trimet.org/alerts/morrisonyamhill/>`_.
```
Comment (ericholscher) [inline docs/conf/portland/2018/news/welcome-to-portland.rst]: I think the construction is diff. this year, and you should be able to get downtown.

File: docs/conf/portland/2018/news/welcome-to-portland.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+* **Riding a bike.** Portland has a `bike share program <https://www.biketownpdx.com/>`_.
+
+Have Fun
+--------
+
+* **Pre-register at the welcome reception.** The meet-and-greet event will be held on Sunday from *6pm-9pm*. You can pick up your badge and enjoy a couple of drinks on us. This is a great chance to meet your fellow documentarians before the conference starts.
```
Comment (ericholscher) [inline docs/conf/portland/2018/news/welcome-to-portland.rst]: Believe we moved this to 5-8.

**Comment by ericholscher** [comment]
Will fix up the notes I mentioned above.

**Pull request merged**